common/connectors-config/src/main/java/com/vmware/connectors/common/utils/CommonUtils.java

```diff
@@ -0,0 +1,32 @@
+package com.vmware.connectors.common.utils;
+
+import com.vmware.connectors.common.payloads.response.Card;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.http.HttpHeaders;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import javax.servlet.http.HttpServletRequest;
+
+/**
+ * Common utility functions to be used across connectors parent.
+ */
+public final class CommonUtils {
+
+    public static final String IMAGE_PATH = "/images/connector.png";
+
+    private CommonUtils() {
+        // Singleton class.
```

Comment:
It's a utility class.  A singleton has exactly 1 instance.  This has no instances (all static methods, no need for an instance).

Reply:
Modified the comment to ```Utility class.```